Template/config/mailgun_id_mapping.php

```diff
@@ -0,0 +1,12 @@
+<h3><img src="<?= $this->url->dir() ?>plugins/Mailgun/mailgun-icon.png"/>&nbsp;<?= t('Mailgun Unknown ID mapping') ?></h3>
+<div class="panel">
+    <?= $this->form->label(t('Set unknown senders to:'), 'MailgunProject_catchall') ?>
```

Comment:
You don't need to add the `:` at the end. This is to stay consistent with the rest.

Reply:
Removed